src/views/Home.vue

```diff
@@ -4,37 +4,71 @@
     p(class="subtitle") BOOKS
     form(class="container-form" @submit.prevent="onSubmit")
       label(class="label-input") First name
-      input(type="text", class="input-primary" v-model="firstName")
+      input(type="text", class="input-primary", v-model="$v.form.firstName.$model")
```

Comment:
use this on all inputs 

```suggestion
      input.input-primary(type="text" v-model="$v.form.firstName")
```